Documentation/Developer/Roundtrip.rst

```diff
@@ -16,7 +16,7 @@ originally created files.
 This is, where "Roundtrip mode" is needed. It aims to preserve your manual changes and applying the new model
 configuration at the same time.
 
-The roundtrip mode is enabled by default. To disable it see the ExtensionBuilder Configuration :doc:`/Configuration/Index`
+The roundtrip mode is enabled by default. To disable it see the Extension Builder Configuration :doc:`/Configuration/Index`
```

Comment:
I'm not sure about this as ExtensionBuilder is the product name

Reply:
ExtensionBuilder has been written as "Extension Builder" before. So the same name should be used everywhere.